prism/src/main/java/org/apache/falcon/resource/AbstractSchedulableEntityManager.java

```diff
@@ -162,11 +162,11 @@ public SchedulableEntityInstanceResult getFeedSLAMissPendingAlerts(String feedNa
             Date end = (endStr == null) ? new Date() : EntityUtil.parseDateUTC(endStr);
 
             if (StringUtils.isBlank(feedName)) {
-                instances.addAll(FeedSLAMonitoringService.get().getFeedSLAMissPendingAlerts(start, end));
+                instances.addAll(EntitySLAMonitoringService.get().getFeedSLAMissPendingAlerts(start, end));
```

Comment:
These changes are fine, but, since there is an API exposed for the FeedSLA, we'll need to extend that API and add/modify corresponding methods to return process SLA misses.